test/karpenter.test.ts

```diff
@@ -73,6 +73,30 @@ describe('Unit tests for Karpenter addon', () => {
         }).toThrow("Consolidation and ttlSecondsAfterEmpty must be mutually exclusive.");
     });
 
+    test("Support Windows2019 as amiFamily", () => {
+        const app = new cdk.App();
+
+        const blueprint = blueprints.EksBlueprint.builder();
+
+        blueprint.account("123567891").region('us-west-1')
+            .version("auto")
+            .addOns(new blueprints.KarpenterAddOn({
```

Comment:
What is this test case accomplishing, if we never build the blueprint? Do you want to add some expectations here, or at least invoke `build` on the blueprint?

Applies elsewhere.